flink-dist/src/main/flink-bin/bin/flink

```diff
@@ -51,5 +51,20 @@ log_setting=(-Dlog.file="$log" -Dlog4j.configuration=file:"$FLINK_CONF_DIR"/log4
 export FLINK_ROOT_DIR
 export FLINK_CONF_DIR
 
+TABLE_JAR_PATH=`echo "$FLINK_ROOT_DIR"/opt/flink-table*.jar`
+libs=("-py " "--python " "-pym " "--py-module ")
+is_python=
+for (( i = 0; i < ${#libs[@]}; i++ )); do
+    if [[ $@ =~ ${libs[i]} ]]; then
+        is_python=1
+        break
+    fi
+done
+if [[ ${is_python} -eq 1 ]]; then
+	TABLE_JAR_PATH=`echo "$FLINK_ROOT_DIR"/opt/flink-table*.jar`
```

Comment:
We should not put the `flink-table` to the classpath by default. Users should do that manually

Reply:
Yes.I have removed these code.